utils/check_copies.py

```diff
@@ -78,6 +78,14 @@
             " {paper_authors}.{supplements}"
         ),
     },
+    "README_ja.md": {
+        "start_prompt": "🤗Transformersは現在、以下のアーキテクチャを提供しています",
+        "end_prompt": "1. 新しいモデルを提供したいですか？",
```

Comment:
```suggestion
        "end_prompt": "1. 新しいモデルを投稿したいですか？",
```
Sorry it's the end prompt the util cannot find. Looks like the line is indeed different in the Japanese README.

Reply:
@sgugger Thanks! There is no problem with the contents of that.